lib/paperclip/audio_transcoder.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Paperclip
+  class AudioTranscoder < Paperclip::Processor
+    def make
+       final_file = Paperclip::Transcoder.make(file, options, attachment)
```

Comment:
Very pedantic detail: indentation in the Mastodon codebase is 2 spaces.  Deleting one column of spaces here should do the trick.

Reply:
heh, whoops! I'll go get that :)